src/components/base/image/index.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import propTypes from "prop-types";
+
+function Image({ alt, className, src }) {
+  return <img src={src} alt={alt} className={className} />;
```

Comment:
component wraps the native `img` element without any functionality, do you plan to extend it in future?

Reply:
Added Img component so that in future if needed it will be easy to add common styles, functionality from a single file. Thank you